cranelift-simplejit/src/backend.rs

```diff
@@ -492,13 +492,13 @@ fn lookup_with_dlsym(name: &str) -> *const u8 {
 
         for handle in &handles {
             let addr = winapi::um::libloaderapi::GetProcAddress(*handle, c_str_ptr);
-            if addr.is_null() {
+            if addr.isnull() {
                 continue;
             }
             return addr as *const u8;
         }
 
-        let msg = if handles[1].is_null() {
+        let msg = if handles[1].isnull() {
```

Comment:
(bad search-and-replace here too; it'd be nice that if there's been new changes cancelling those, they'd be included in this same commit instead, to make each commit atomic and testable independently from the others, which is quite useful when using `git bisect` for instance.)

Reply:
I've decided to squash all the commits together. I initially separated it into smaller commits so that it would be easier to follow the changes, but it probably makes more sense it have it be one commit overall for merging.